addon/components/ember-flatpickr.js

```diff
@@ -22,6 +23,8 @@ export default Component.extend({
 
     // Pass all values and setup flatpickr
     run.scheduleOnce('afterRender', this, function() {
+      const fastboot = getOwner(this).lookup('service:fastboot');
```

Comment:
Rather than using `getOwner`, can we just do `fastboot: service()` up top? Then check `this.get('fastboot.isFastBoot')`?

Reply:
I think `fastboot: Ember.inject.service()` will fail for non-fastboot apps.

Reply:
Ah okay. That makes sense. I probably need to fix that in some other repos 😬